Aquí va...

xavidp opened the pull request:
Hola de nuevo, a ver si ahora ya sí que esto avanza.

Van los primeros tres tutoriales traducidos, y algunos retoques a los archivos del proyecto (rutas corregidas, gitignore actualizado, etc).

En otro commit posterior haré otro pull request del `renv`

xavidp committed:
Actualizado gitignore original para que ignore los Rproj locales de los contribuidores (para evitar nombres antiguos o problemas por usar diferentes plataformas, LineEndingConversion, etc). Por si hay gente que contribuye des de no-Posix, que no es mi caso, ahora mismo, pero hay mucha gente que aún usa esos OS's, etc., y mejor evitar problemas cuando quieran contribuir desde allí

xavidp committed:
Primeros tres tutoriales traducidos y elimino el archivo placeholder que se dejço en la versión inicial para que existiese la carpeta donde poner los tutoriales, etc. Gracias Riva

xavidp committed:
elimino archivo placeholder inicial y pongo un mini-Readme en su sitio

xavidp committed:
corregidas rutas diversas y nombre

xavidp committed:
renv activado

pachadotdev commented:
@xavidp  gràcies!
ara com ho fem per que gh-actions funcioni amb els canvis a netlify

xavidp commented:
> @xavidp gràcies! ara com ho fem per que gh-actions funcioni amb els canvis a netlify

ni idea, no domino esta parte (i no sabia que parlaves català!!! 8-)

xavidp commented:
netlify-related...., alucino, si tenéis hecha una sección de respuestas a los ejercicios del libro!
https://r4ds-soluciones.netlify.app/

¡Y veo que eres de Chile! (tengo una hermana en Santiago)

Thumbs-up

pachadotdev commented:
> ¡Y veo que eres de Chile! (tengo una hermana en Santiago)

si, pero no vivo en Chile hace mucho tiempo

tenemos las rptas, ahora tenemos que arreglar los problemas que hay con netlify y gh-actions

xavidp committed:
4o tutorial traducido: Resumir Datos

https://github.com/rstudio/learnr/blob/main/inst/tutorials/ex-data-summarise/ex-data-manip-summarise.Rmd

xavidp committed:
Tutorial de puesta a punto traducido (el texto), pero se siguen usando unos videos in inglés  (en vimeo) sobre como instalar R, RStudio y paquetes de R

pachadotdev committed:
Merge branch 'main' into master
